tests/unit/translator/resources/decision.pr

```diff
@@ -0,0 +1,84 @@
+
+--  Header file for function ComplexSdl in SDL language
+--  Component type : FALSE
+--  Instance of :
+--  Generated by TASTE on 2022-01-13 19:10:46
+--  Context Parameters present : NO
+--  Provided interfaces : impulse
+--  Required interfaces : response
+--  User-defined properties for this function:
+--     * TASTE::Is_Component_Type = false
+--     * TASTE_IV_Properties::FPGA_Configurations = AllModes
+--  DO NOT EDIT THIS FILE, IT WILL BE OVERWRITTEN DURING THE BUILD
+/* CIF Keep Specific Geode ASNFilename 'dataview-uniq.asn' */
+use Datamodel;
+
+system Complexsdl;
+
+   /* CIF Keep Specific Geode PARAMNAMES x_in */
+   signal impulse (MyInteger);
+
+   /* CIF Keep Specific Geode PARAMNAMES x_in */
+   signal response (MyInteger);
+
+   channel c
+      from env to Complexsdl with impulse;
+      from Complexsdl to env with response;
+   endchannel;
+
+   block Complexsdl;
+
+      signalroute r
+      from env to Complexsdl with impulse;
+      from Complexsdl to env with response;
+      connect c and r;
+
+   /* CIF PROCESS (250, 150), (150, 75) */
+process Complexsdl;
+    /* CIF TEXT (1202, 0), (267, 140) */
+    -- Text area for declarations and comments
+
+    DCL tmp MyInteger;
+    DCL result MyInteger;
+    /* CIF ENDTEXT */
+    /* CIF START (320, 21), (70, 35) */
+    START;
+        /* CIF NEXTSTATE (320, 71), (70, 35) */
+        NEXTSTATE Wait;
+    /* CIF state (450, 21), (70, 35) */
+    state Wait;
+        /* CIF input (434, 76), (101, 35) */
+        input impulse(tmp);
+            /* CIF decision (450, 131), (70, 50) */
+            decision tmp;
+                /* CIF ANSWER (682, 392), (70, 23) */
+                (0):
```

Comment:
Do we have a test which checks e.g. answer x > 10 or similar?

Reply:
Such test was created with initial implementation of decision. expression_based_decision.pr and value_based_decision.pr.